db/db-sqlx/src/page.rs

```diff
@@ -0,0 +1,51 @@
+use super::*;
+
+pub(crate) const COUNT_COLUMN: &str = "count";
+
+pub fn build_query_page_sql(
```

Comment:
I think this fn should be designed to accept a mutable owned query builder since we will finish the query build process in this fn, it will simplified the code:

```
pub fn build_query_page_sql(
    mut qb: SqlBuilder,
    pagination: &PaginationRequest,
) -> Result<(String, String)> {
    if let Some(id) = pagination.cursor {
        let id = i64::try_from(id).unwrap_or(i64::MAX);
        match pagination.order {
            Order::Asc => {
                qb.and_where_gt("id", id);
            }
            Order::Desc => {
                qb.and_where_lt("id", id);
            }
        }
    }

let sql_sub_query = query_builder.subquery()?;
    match pagination.order {
        Order::Asc => {
            qb.order_by("id", false);
        }
        Order::Desc => {
            qb.order_by("id", true);
        }
    }

   ...
```

this suggestion is also applied to other SqlBuilder related code.